diff/schema_list_diff.go

```diff
@@ -13,9 +15,11 @@ The result is a combination of two diffs:
 2. Diff of schemas without a $ref (inline schemas): number of added/deleted schemas; modified=only if exactly one schema was added and one deleted, the Modified field will show a diff between them
 */
 type SchemaListDiff struct {
-	Added    int             `json:"added,omitempty" yaml:"added,omitempty"`
-	Deleted  int             `json:"deleted,omitempty" yaml:"deleted,omitempty"`
-	Modified ModifiedSchemas `json:"modified,omitempty" yaml:"modified,omitempty"`
+	Added          int              `json:"added,omitempty" yaml:"added,omitempty"`
```

Comment:
I recommend to remove Added and Deleted properties from this struct.
We can use len(AddedSchemas) and len(DeletedSchemas) instead.

Reply:
I was thinking initially about this option but didn't want to introduce a breaking change int the oasdiff output (changing Added from int to a string slice), but I can definitely make the change if you agree with it

Reply:
Let's do it.

Reply:
done, can you please check the last commit?